src/OrchardCore.Modules/OrchardCore.ContentFields/Settings/MultiSelectFieldSettings.cs

```diff
@@ -0,0 +1,23 @@
+using Newtonsoft.Json;
+
+namespace OrchardCore.ContentFields.Settings
+{
+    public class MultiSelectFieldSettings
+    {
+        public string Hint { get; set; }
+        public bool Required { get; set; }
+        public string Options { get; set; }
```

Comment:
This should just be a `List<MultiSelectListValueOption`. Deserialized once in the driver, and stored on the settings.

You may need a ViewModel